eng/build.ps1

```diff
@@ -581,6 +581,9 @@ function Deploy-VsixViaTool() {
   # Disable background download UI to avoid toasts
   &$vsRegEdit set "$vsDir" $hive HKCU "FeatureFlags\Setup\BackgroundDownload" Value dword 0
 
+  # Disable text spell checker to avoid spurious warnings in the error list
```

Comment:
Can we or should we update our tests instead to filter out the spell checker issues? Or fix the bad spelling if possible? I'd be worried if we might potentially mask bugs where we're both running at the same time.

Reply:
What version was the spell checker added in? I'm not sure the CI machines have it yet. We can file a follow-up issue if it's a goal to have both running at the same time in tests, provided there are APIs available to wait for that feature to complete its background work before making assertions against the outcome.